package.json

```diff
@@ -10,7 +10,8 @@
         "build": "yarn workspaces foreach -Ap --topological-dev run build",
         "build:libraries": "yarn workspaces foreach -Apt --no-private run build",
         "build:subgraph": "yarn workspace semaphore-subgraph build",
-        "compile:contracts": "yarn workspace semaphore-contracts compile",
+        "copy:abi": "cp  packages/contracts/artifacts/contracts/Semaphore.sol/Semaphore.json packages/utils/src/semaphore-abi.json",
```

Comment:
`cp` will only work on Unix-like systems. A Node package alternative like [`ncp`](https://www.npmjs.com/package/ncp) should be better here.

I'd also suggest moving this script (`copy:abi`) to the `packages/contracts/package.json` file directly, otherwise if I run `yarn compile` from `packages/contracts` this script won't be executed.

Reply:
`semaphore-abi.json` actually includes bytecode and other data. `semaphore-interface.json` should be more representative.